types/protobuf.go

```diff
@@ -53,10 +53,7 @@ func (tm2pb) ValidatorUpdate(val *Validator) abci.ValidatorUpdate {
 		NodeAddress: val.NodeAddress.String(),
 	}
 	if val.PubKey != nil {
-		pk, err := cryptoenc.PubKeyToProto(val.PubKey)
-		if err != nil {
-			panic(err)
-		}
+		pk := cryptoenc.MustPubKeyToProto(val.PubKey)
```

Comment:
I know this is not part of this PR, but I think we need proper error handling in the main code, and only allow panics in test code.

Reply:
I agree, but it's better to do it in the task of refactoring unnecessary `panic`